src/utils/validateTx.ts

```diff
@@ -1,19 +1,29 @@
 import { getTx } from '../web3/ether';
 import { TxData } from '../../types';
+// import Web3 from 'web3';
```

Comment:
use import

Reply:
There is some issue we are working on it!